test/integration/targets/vmware_tag/tasks/tag_manager_ops.yml

```diff
@@ -0,0 +1,45 @@
+# Test code for the vmware_tag Manager Operations.
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- when: vcsim is not defined
+  block:
+
+    - name: get a list of VMS from vcsim
+      uri:
+        url: http://{{ vcsim }}:5000/govc_find?filter=VM
```

Comment:
vcsim is not defined so I think we need to get list of VM using `vmware_vm_facts`